File: rei-do-chopp-site/src/app/orders/models/requests/order-additional-fee-list.request.ts
import { PaginationRequest } from "../../../shared/models/requests/pagination.request";

export class OrderAdditionalFeeListRequest extends PaginationRequest{
  OrderId: number;

  constructor(params: Partial<OrderAdditionalFeeListRequest>) {
    super(params);
    this.OrderId = params.OrderId || null;
  }
}

File: rei-do-chopp-api/ReiDoChopp.DataTransfer/OrderAdditionalFees/Requests/OrderAdditionalFeeListRequest.cs
using System;
using ReiDoChopp.Domain.Utils.Enums;
using ReiDoChopp.DataTransfer.Pagination.Responses;
using ReiDoChopp.DataTransfer.Pagination.Requests;

namespace ReiDoChopp.DataTransfer.OrderAdditionalFees.Request
{
    public class OrderAdditionalFeeListRequest : PaginationRequest
    {
        public int? Id { get; set; }
        public int? OrderId { get; set; }
        public double? Value { get; set; }
        public string Description { get; set; }


        public OrderAdditionalFeeListRequest() : base(pageSize: 10, ordenationField: "Id", ordenationType: OrdenationTypeEnum.ASC) {}
    }
}